Lib/asyncio/tasks.py

```diff
@@ -217,14 +217,12 @@ def cancel(self, msg=None):
         return True
 
     def cancelling(self):
```

Comment:
Is the `cancelling()` the best name for returning the number or cancel requests?
It was good for bool flag, a counter needs better name IMHO.

Reply:
Hm, do you have a suggestion? I was thinking that it's still usable as a "truthy" value. Usually when you're interested in the cancel count you should call `uncancel()` and use its return value.

Reply:
I have no good answer.
`cancel_requests_count()` describes exact meaning but the name is too long.
`cancelling()` can return bool but it looks like procrastination: why return less info than we really have.
The third option is dropping `cancelling()` method entirely, it is not required by the current asyncio code.
We can consider adding the method later on-demand.

Reply:
Note: `.cancelling()` doesn't exist in Python 3.10

Reply:
It's used in TaskGroup currently, so if we were to delete it we'd have to rewrite the code there in this same PR. It's also used by `Task.__repr__()` (via `_task_repr_info()`). But of those will work even though we upgraded it from a bool to an int. I'd say let's keep it for now, if we regret it we can rename or remove it before beta 1.

Reply:
Deal!